src/controllers/commands/CommandController.cpp

```diff
@@ -878,23 +878,79 @@ void CommandController::initialize(Settings &, Paths &paths)
         return "";
     });
 
-    this->registerCommand(
-        "/chatters", [](const auto & /*words*/, auto channel) {
-            auto twitchChannel = dynamic_cast<TwitchChannel *>(channel.get());
+    this->registerCommand("/chatters", [](const auto &words, auto channel) {
+        auto formatError = [](HelixGetChattersError error, QString message) {
+            using Error = HelixGetChattersError;
 
-            if (twitchChannel == nullptr)
+            QString errorMessage = QString("Failed to get chatter count: ");
+
+            switch (error)
             {
-                channel->addMessage(makeSystemMessage(
-                    "The /chatters command only works in Twitch Channels"));
-                return "";
+                case Error::Forwarded: {
+                    errorMessage += message;
+                }
+                break;
+
+                case Error::UserMissingScope: {
+                    errorMessage += "Missing required scope. "
+                                    "Re-login with your "
+                                    "account and try again.";
+                }
+                break;
+
+                case Error::UserNotAuthorized: {
+                    errorMessage += "You don't have permission to "
+                                    "perform that action.";
+                }
+                break;
+
+                case Error::Unknown: {
+                    errorMessage += "An unknown error has occurred.";
+                }
+                break;
             }
+            return errorMessage;
+        };
 
+        if (words.size() != 1)
+        {
+            channel->addMessage(makeSystemMessage("Usage: /chatters"));
+            return "";
+        }
```

Comment:
Why do we care about the input for this command?

Reply:
I was just following the convention of the other commands, but I guess those were using arguments and could lead being misused. I don't mind removing it.

Reply:
nvm, it was already removed